editors/code/package.json

```diff
@@ -266,6 +273,13 @@
                     "default": "check",
                     "markdownDescription": "Cargo command to use for `cargo check`"
                 },
+                "rust-analyzer.checkOnSave.overrideCommand": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    },
+                    "markdownDescription": "Advanced option, fully override the command rust-analyzer uses for checking. The command must include `--message=format=json`."
```

Comment:
Is this actually a requirement? Or do you just need JSON output?

Reply:
We need json output, which typically means `--message=format=json`. I guess I can phrase this better